app/src/component/walletSend/view.js

```diff
@@ -52,8 +52,17 @@ class WalletSend extends React.PureComponent<Props> {
     }
 
     if (amount && address) {
-      sendToAddress(address, parseFloat(amount));
-      this.setState({ address: null, amount: null });
+      // Show confirmation before send
+      Alert.alert(
+      'Send LBC',
+      `Are you sure you want to send ${amount} LBC to ${address}?`,
+      [
+        { text: 'No' },
+        { text: 'Yes', onPress: () => {
+          sendToAddress(address, parseFloat(amount));
+          this.setState({ address: null, amount: null });
```

Comment:
Out of scope for this PR, but the address/amount should only be cleared if the tx is successful. Then users don't have to re-add all of the info if something went wrong.

Reply:
Yeah, that makes sense. Will address this in the future.